packages/snap-store-mobx/src/Meta/README.md

```diff
@@ -0,0 +1,46 @@
+# MetaStore
+
+Each root store constructs a MetaStore to hold meta information about a site. 
```

Comment:
Can we briefly mention what a root store is, or point to that information if it exists elsewhere?

Reply:
Perhaps also mention the purpose is to contain the response from the Searchspring meta API which includes information about site configuration and feature settings.